### ISSUE TITLE
Using the Error ERRRREQUIRE-ESM

### ISSUE THREAD

**Issue by Riesenzahn** [Clarkkkk/workbox-rspack-plugin#4]
### dependencies
 
```package.json
  "devDependencies": {
    "@aaroon/workbox-rspack-plugin": "^0.3.0",
    "@release-it/conventional-changelog": "^8.0.1",
    "@rspack/cli": "0.5.9",
    "@rspack/core": "0.5.9",
    "@rspack/plugin-minify": "^0.5.9",
    "@rspack/plugin-react-refresh": "0.5.9",
    "tailwindcss": "^3.4.3",
    "ts-node": "^10.9.2",
    "typescript": "^5.4.3",
    "workbox-rspack-plugin": "^0.0.2"
  }
  ```
  
### commit

  [[e956732ba320cdb25440714ae9aba5c485b57f41](test: swr)](https://github.com/h7ml/shark/commit/e956732ba320cdb25440714ae9aba5c485b57f41)
  
 ### log
 
 

> xxx\code\github\h7ml\shark>pnpm build
rspack-plugin@0.3.0_@rspack+core@0.5.9\node_modules\@aaroon\workbox-rspack-plugin\dist\index.cjs not supported.
Instead change the require of index.js in xxx\code\github\h7ml\shark\node_modules\.pnpm\@aaroon+workbox-rspack-plugin@0.3.0_@rspack+core@0.5.9\node_modules\@aaroon\workbox-rspack-plugin\dist\index.cjs to a dynamic import() which is available in all CommonJS modules.
    at require.extensions.<computed> [as .js] (xxx\code\github\h7ml\shark\node_modules\.pnpm\ts-node@10.9.2_@types+node@20.11.30_typescript@5.4.3\node_modules\ts-node\dist\index.js:851:20)
    at Module.<anonymous> (xxx\code\github\h7ml\shark\node_modules\.pnpm\@aaroon+workbox-rspack-plugin@0.3.0_@rspack+core@0.5.9\node_modules\@aaroon\workbox-rspack-plugin\dist\index.cjs:1:283)
    at require.extensions.<computed> [as .js] (xxx\code\github\h7ml\shark\node_modules\.pnpm\ts-node@10.9.2_@types+node@20.11.30_typescript@5.4.3\node_modules\ts-node\dist\index.js:851:20) {
  code: 'ERR_REQUIRE_ESM'
}
 ELIFECYCLE  Command failed with exit code 1.

**Comment by Riesenzahn** [comment]
Additionally, the document https://github.com/Clarkkkk/workbox-rspack-plugin/blob/main/README.md?plain=1#L58 This line uses workbox rspack plugin instead of @ aaroon/workbox rspack plugin. Baby, is this an error in the document.

**Comment by Clarkkkk** [comment]
Thanks, fixed in 0.3.1.
